src/config/software-documentation/index.md

```diff
@@ -0,0 +1,13 @@
+# Software documentation
+
+Most common format of documentation in Void Linux are [manual pages](./man.md).
+
+Many programs are provided along with documentation in other formats, like HTML.
+Usually it is located in `/usr/share/doc/<program>` directory.
+
+Most extensive documentation may be split into separate `*-doc` package, like
+`julia-doc`. This is often case for programming languages, databases and big
+software libraries.
+
+Software that need configuration specific to Void may include
+`/usr/share/doc/<package>/README.voidlinux` document describing it.
```

Comment:
```suggestion
Software that requires configuration specific to Void will include
`/usr/share/doc/<package>/README.voidlinux` document describing it.
```

I changed it to `will` as kind of a promise that we *will* document all Void special cases. If you don't agree with the idea or don't think it fits, feel free to remove.

That said, I don't love this paragraph. First is how it's worded, but I haven't yet figured a better way; but also that not all READMEs will be only for void specific information, since they might just include helpful information on the initial setup for some piece of software.

Reply:
Rewritten.